client/packages/common/src/intl/locales/en/common.json

```diff
@@ -37,6 +37,7 @@
   "button.sync-now": "Sync now!",
   "button.try-again": "Try again",
   "button.view": "View",
+  "button.zero-line-quantity": "0 line quantity",
```

Comment:
Doesn't say much what it does or read like a command... Maybe something like "Set quantities to 0"?